### PR TITLE
Add package: e

### PR DESCRIPTION
**Pull request by matthiasbeyer**
`e` is a minimalistic, dependency free commandline calculator.

### CODE REVIEW

File: pkgs/tools/misc/e/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  src = fetchurl {
+    url = "http://www.sourcefiles.org/Productivity_Tools/Calculators/e-${version}.tar.gz";
+    sha256 = "1rxkb0wm136smdn0q359srvw0ppah06441rla7hy1h0k9n0bwawk";
+  };
+
+  buildInputs = [ stdenv ];
+
+  # These morons ship a binary in their tar.gz. What the hell?
+  buildPhase = ''
+    rm e
```
Comment (peti) [inline pkgs/tools/misc/e/default.nix]: You should probably write this is as `preBuild = "rm e";`. Then you can leave the `buildPhase` attribute alone.

File: pkgs/tools/misc/e/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  name = "e-${version}";
+  
+  src = fetchurl {
+    url = "http://www.sourcefiles.org/Productivity_Tools/Calculators/e-${version}.tar.gz";
+    sha256 = "1rxkb0wm136smdn0q359srvw0ppah06441rla7hy1h0k9n0bwawk";
+  };
+
+  buildInputs = [ stdenv ];
+
+  # These morons ship a binary in their tar.gz. What the hell?
```
Comment (peti) [inline pkgs/tools/misc/e/default.nix]: That was probably an accident. You might want to notify upstream.

**Comment by matthiasbeyer** [comment]
I guess this should be closed unmerged. I just noticed that "upstream" is a collection of source code archives and not a site where a developer hosted her/his source code.

So it is rather unclear what is the real "upstream" here.

**Comment by MP2E** [comment]
ok closing at author's request (note that you can close your own issues as well)

**Pull request closed without merging**